AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -532,7 +540,7 @@ + (void)registerSharedApplicationNotifications
   [center addObserver:self selector:@selector(willEnterForeground:) name:UIApplicationWillEnterForegroundNotification object:nil];
 }
 
-static ASLayoutRangeMode __rangeModeForMemoryWarnings = ASLayoutRangeModeVisibleOnly;
+static ASLayoutRangeMode __rangeModeForMemoryWarnings = ASLayoutRangeModeLowMemory;
```

Comment:
Could you file a 2.1 or later task to revisit this?  The ideal behavior would be VisibleOnly for things with visibilityDistance of 0 or 1, but to use Low Memory if visibilityDistance >= 2 — but then, once something becomes 2 -> 1, it needs to automatically trigger going from LowMemory to VisibleOnly.